config.go

```diff
@@ -43,7 +43,7 @@ const (
 	defaultBlockMaxSize      = 375000
 	blockMaxSizeMin          = 1000
 	blockMaxSizeMax          = wire.MaxBlockPayload - 1000
-	defaultBlockPrioritySize = 50000
+	defaultBlockPrioritySize = 20000
```

Comment:
Yes. The code was updated in accordance with comments from davecgh. Now by default it allows 20kb for high priority regular transactions and revocations, while sorting tickets by ticket fees. This ensures that high fee tickets as well as high priority transactions are included in a block if blocks are full.